tools/management/commands/upload_excel_bias.py

```diff
@@ -9,13 +9,11 @@
 from common.tools import fetch_from_cache, save_to_cache, fetch_from_web_api
 from residue.models import Residue
 from protein.models import Protein
-from ligand.models import *
+from ligand.models import BiasedExperiment, BiasedExperimentVendors, ExperimentAssay, ExperimentAssayAuthors, Ligand, LigandProperities, LigandType, LigandVendorLink
```

Comment:
You're still missing AnalyzedExperiment here.

Reply:
Done